### DISCUSSION TITLE
unable to read with readfile

### DISCUSSION THREAD

**Discussion by kapilpjain1** [exceljs/exceljs#2497] [Q&A]
If my excel file has 3/4 sheets same code reads excel file correctly
But when size of sheets in excel file goes to 20 
it doesnt read excel file
Code doesnt read sheet

How is it possible
How is it possible to read plenty sheets excel file easily
How to use Stream API to read multiple sheets excel file  in excelJS
Is stream API different than ExcelJS or part of it
Online documentation doesnt clear it

**Comment by vchouhan** [comment]
@kapilpjain1 can you please share your code and your excel file.
